### DISCUSSION TITLE
使用java获取 群里面的成员活跃 报错

### DISCUSSION THREAD

**Discussion by yujumpjump** [mamoe/mirai#2523] [Q&A]
报错内容：love.forte.simbot.event.EventListenerProcessingException: kotlinx.serialization.MissingFieldException: Field 'ginfo' is required for type with serial name 'net.mamoe.mirai.internal.contact.active.GroupActiveData', but it was missing at path: $

我想是否是一个bug

**Comment by cssxsh** [comment]
提交错误是请附上完整的日志
日志文件在 logs 文件夹

**Reply by yujumpjump** [reply]
我使用的是simbot3开发的 不知道具体的日志文件在那个地方
